src/gui/widgets/FadeButton.cpp

```diff
@@ -21,7 +21,7 @@
  * Boston, MA 02110-1301 USA.
  *
  */
- 
+
```

Comment:
Irrelevant to this PR. 
```suggestion
 
```

Reply:
My IDE automatically removes spaces at ends of lines, and it's probably the best practice. Is it better to leave them as-is?

Reply:
We want to keep the diff as clean as possible.

Reply:
The spaces at ends of lines should be removed from a technical standpoint, and unless it's done as part of other PRs its probably never going to happen (because is it really worth it to make a separate PR just to remove spaces?)

Reply:
Just for information, if we really decide to run clang-format over the whole code, all whitespace will be fixed. In this case, there will indeed be a *separate PR just to remove spaces*.

Reply:
Ok.